src/dndbeyond/base/utils.js

```diff
@@ -305,8 +305,8 @@ function applyGWFIfRequired(action_name, properties, damage) {
         (
             (properties["Properties"].includes("Versatile") && character.getSetting("versatile-choice") != "one") || 
             properties["Properties"].includes("Two-Handed"))) ||
-            action_name == "Polearm Master - Bonus Attack" ||
-            action_name == "Pole Strike")
+            action_name.includes("Polearm Master") && character.hasFeat("Polearm Master", false)  ||
+            action_name.includes("Pole Strike") && character.hasFeat("Polearm Master 2024", false))
```

Comment:
due to logical operator precedence and mixing of `&&` and `||`, I think this check needs parenthesis for more clarity/maintainability. I'll add them as I push